addons/mail/static/src/models/messaging_notification_handler/messaging_notification_handler.js

```diff
@@ -266,8 +266,8 @@ function factory(dependencies) {
                     channel.markAsFetched();
                 }
                 // (re)open chat on receiving new message
-                if (channel.channel_type !== 'channel' && !this.env.messaging.device.isMobile) {
-                    this.env.messaging.chatWindowManager.openThread(channel);
+                if (channel.channel_type !== 'channel' && !this.env.messaging.device.isMobile && !channel.chatWindow) {
+                    this.env.messaging.chatWindowManager.openThread(channel, { isFolded: channel.chatWindowIsFolded });
```

Comment:
The change on this line can be removed then, because `chatWindowIsFolded` will always be undefined if `chatWindow` is not set.